src/app/pages/signing/signing.component.ts

```diff
@@ -19,6 +19,9 @@ import { Title } from '@angular/platform-browser';
 export class SigningComponent implements OnInit {
   public name = 'your friend';
   public sending = false;
+  public embedLink = 'https://bit.ly/3de5LUs';
```

Comment:
Is Arvind okay with his picture being the default? I know I wouldn't want my picture there lol.

Reply:
```suggestion
  public embedLink = '';
```